common/supplemental/supplementalMetadata.xml

```diff
@@ -1572,11 +1585,8 @@ For terms of use, see http://www.unicode.org/copyright.html
 						
 			
             <!-- variants -->
-            <variantAlias type="AALAND" replacement="AX" reason="deprecated"/>
-            <variantAlias type="POLYTONI" replacement="POLYTON" reason="deprecated"/>
-            <variantAlias type="HEPLOC" replacement="ALALC97" reason="deprecated"/>
-			<variantAlias type="arevela" replacement="hy" reason="deprecated"/> <!-- arevela -->
-			<variantAlias type="arevmda" replacement="hyw" reason="deprecated"/> <!-- arevmda -->
+            <variantAlias type="aaland" replacement="AX" reason="deprecated"/>
```

Comment:
About <variantAlias type="aaland" replacement="AX" reason="deprecated"/>
how could the code figure the AX not not a language code "ax" but a region code "AX" here. 
Should this be <variantAlias type="aaland" replacement="und-AX" reason="deprecated"/>
instead?

Reply:
Yes, that would be better.

Reply:
fixed.